region/crateria/east.json

```diff
@@ -636,22 +865,40 @@
           "to": [
             {
               "id": 10,
-              "requires": [ "Morph" ]
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [ "Morph" ]
+                }
+              ]
             },
             {
               "id": 12,
-              "requires": [
-                "Super",
-                {"ammo": {
-                  "type": "Super",
-                  "count": 1
-                }}
-              ],
-              "note": "Super block respawns"
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    "Super",
+                    {"ammo": {
+                      "type": "Super",
+                      "count": 1
+                    }}
+                  ],
+                  "note": "Super is for a Super Block. It's not an obstacle because it respawns."
```

Comment:
This doesn't represent that you only need 1 super to get the item and get back out.  not that its trivial to do

Reply:
Indeed. I don't know how to fit it into the structure and I don't think the logical gains would be worth the effort, but I'll add a note.

Reply:
Or I guess it could just be a notable strat. That works by accident as long as we don't find a new way to access node 14.